test/models/user_test.rb

```diff
@@ -26,6 +26,28 @@
     end
   end
 
+  describe "#time_format" do
+  let(:user) { User.create!(name: "jimbob", email: 'test@test.com') }
+    it "should have a default time format of relative" do
+      user.time_format.must_equal('relative')
+    end
+
+    it "should not update" do
```

Comment:
try to write the `it` in a way that ideally makes it unnecessary to read what the test does

`it "does not update to a invalid time format"`